pom.xml

```diff
@@ -226,7 +226,7 @@
     <wildfly-maven-plugin.version>1.2.0.Final</wildfly-maven-plugin.version>
 
     <aerogear.bom.version>1.1.15</aerogear.bom.version>
-    <admin-ui.version>2.2.4</admin-ui.version>
+    <admin-ui.version>2.2.5</admin-ui.version>
```

Comment:
Latest published version in maven repo is 2.2.1 (https://search.maven.org/artifact/org.aerogear.unifiedpush/admin-ui)

Reply:
It is hosted in jboss's repository.https://repository.jboss.org/nexus/service/local/repositories/thirdparty-releases/content/org/aerogear/unifiedpush/admin-ui/